projects/core/src/user/facade/user.service.ts

```diff
@@ -37,28 +38,10 @@ export class UserService {
   /**
    * Register a new user
    *
-   * @param titleCode a title code
-   * @param firstName first name
-   * @param lastName last name
-   * @param email an email
-   * @param password a password
+   * @param submitFormData as UserRegisterFormData
    */
-  register(
-    titleCode: string,
-    firstName: string,
-    lastName: string,
-    email: string,
-    password: string
-  ): void {
-    this.store.dispatch(
-      new fromStore.RegisterUser({
-        firstName: firstName,
-        lastName: lastName,
-        password: password,
-        titleCode: titleCode,
-        uid: email
-      })
-    );
+  register(submitFormData: UserRegisterFormData) {
```

Comment:
`submitFormData` is imho badly named. let's change it to something more semantic